### DISCUSSION TITLE
Disable verbose

### DISCUSSION THREAD

**Discussion by EdoAlvarezR** [SMTorg/smt#676] [Q&A] [answered]
Hi,

Is there a way of disabling the verbose message (`Predicting ...`) every time that `sm.predict_values` is called to evaluate the model? I'm using an KRG model.

_-Ed Alvarez_

**Comment by relf** [comment, marked as answer]
Hi. To disable printing, you can pass `print_global=False`:
```python
sm = KRG(print_global=False)
```

 See [`print_*` options](https://smt.readthedocs.io/en/stable/_src_docs/surrogate_models/gpr/krg.html#options) for more control.
